flake.nix

```diff
@@ -1,15 +1,27 @@
 {
   inputs = {
-    common.url = "github:nammayatri/common";
-
+    # Note: Note: change to upstream when upstream merged
+    # common.url = "github:nammayatri/common";
+    common.url = "github:arjunkathuria/common/Mobility-GHC927-Rebased";
+    nixpkgs.follows = "common/nixpkgs";
+    flake-parts.follows = "common/flake-parts";
```

Comment:
What are all these `follows` for? I see it in the other two repos as well, along an extraneous `systems` input.